addons/mail/static/src/js/client_action.js

```diff
@@ -430,6 +536,9 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
         this.thread.on('toggle_star_status', this, function (message_id) {
             chat_manager.toggle_star_status(message_id);
         });
+        this.thread.on('moderate', this, this._onModeration);
```

Comment:
wrong mechanism. this should be done by using trigger_up in thread widget. remove these 3 lines